### ISSUE TITLE
Does it still work?

### ISSUE THREAD

**Issue by andi-gashi** [dgalli1/ProxerToMyAnimeList#1]
Received error messages.

**Comment by dgalli1** [comment]
I don't think so i didn't try it for a long time, and asfar as i know MAL changed their API. The parser for Proxer should still work tho. Maybe only the dependencies need a update

**Comment by andi-gashi** [comment]
Ok, sadly I'm not code-savvy enough to do that. But thank you anyways :)
